vendor/assets/javascripts/components/accordion.js

```diff
@@ -157,9 +163,9 @@
         } else {
 
             var tmp = {
-                position   : $ele.css("position"),
-                visibility : $ele.css("visibility"),
-                display    : $ele.css("display")
+                position   : $ele.css('position'),
```

Comment:
whereas I agree that this is a preferred format, consider making linter PR's directly to YooTheme's core source for the lib